packages/sc-deployer/src/utils/maxCoins.ts

```diff
@@ -1,8 +1,8 @@
 import { fromMAS } from '@massalabs/massa-web3';
 import { ISCData } from '../interfaces';
 
-const BASE_ACOUNT_CREATION_COST = fromMAS(0.01);
-const PRICE_PER_BYTE = fromMAS(0.001);
+const BASE_ACOUNT_CREATION_COST = fromMAS(0.001);
+const PRICE_PER_BYTE = fromMAS(0.0001);
 
```

Comment:
Another constant is needed, not sure its the good place to put it. but it as to be somewhere:
its the fixed cost of creation for a new ledger entry wich is equal to 4 * PRICE_PER_BYTE